networks.json

```diff
@@ -35,8 +59,34 @@
       "transactionHash": "0x8737379f9eb9e8e98aa77b74f46b40122770c96b7e87d85d37d18414166e5d49"
     },
     "4": {
-      "address": "0x576B76eebE6B5411c0ef310E65De9Bff8A60130F",
-      "transactionHash": "0x2adf9ab9541091ff2ea1035247fb4b5fa49c847018f4ce96c53657cbb0c13a67"
+      "address": "0x17174dC1b62add32a1DE477A357e75b0dcDEed6E",
+      "transactionHash": "0x02a4080fc1c72e4aae4e970740012315a6cbc9509a8d0b1e683fa6b03c49a79a"
+    },
+    "77": {
+      "address": "0xa57EBD93faa73b3491aAe396557D6ceC24fC6984",
+      "transactionHash": "0x359da6ef46339b7f17ee9525f8dc8cbb93e9a1b234c650c8dafd105b2404686c"
+    },
+    "100": {
+      "address": "0x2bf1BFb0eB6276a4F4B60044068Cb8CdEB89f79B",
+      "transactionHash": "0xc57922858be91b2e641d2ae57d44975b861853d384fde4a12032a04a8d0b72d7"
+    }
+  },
+  "RealitioScalarAdapter": {
+    "1": {
+      "address": "0xaa548EfBb0972e0c4b9551dcCfb6B787A1B90082",
+      "transactionHash": "0x781e582a3476bd59d35faa6c47164e5e51f0356a6756a7fcc5fbc7a79d4eed58"
+    },
+    "4": {
+      "address": "0x0e8Db8caD541C0Bf5b611636e81fEc0828bc7902",
+      "transactionHash": "0xbb128a058cd4a3ca7535da2a646ec813a2e9362e48a8b8d447e9f09007ef441b"
+    },
+    "77": {
+      "address": "0x86459E9eA6cF4caEe9F8F4cb1203d38EaB3cbD34",
+      "transactionHash": "0xd18fa0bd6cfa38a157641688956965d60895d85d92971df159ee3982a16bef5e"
+    },
+    "100": {
+      "address": "0xb97FCb6adf4c4aF9981932a004e6CC47173d0Bfc",
+      "transactionHash": "0x07bb34b2e09a899fc7914638b424e2fbd7819401ca5a083c75b55dca47ba83d4"
     }
   },
   "RealitioScalarAdapter": {
```

Comment:
Looks like we have `RealitioScalarAdapter` listed twice here.

Reply:
great catch @davidalbela is preparing a new PR!